articles/integrations/aws.md

```diff
@@ -271,7 +271,7 @@ The result of calling the delegation endpoint will contain the AWS token in the
 }
 ```
 
-**NOTE:** The Auth0 client libraries simplify calling these endpoints. See our [GitHub repo](https://github.com/auth0/) for the latest SDKs. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
+**NOTE:** The [Auth0 client libraries](/libraries) simplify the process of calling these endpoints. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
```

Comment:
@khorne3 The _Delegation Token Request_ does not exist.

@jeffreylees Can you help out on what Katie should use here?